components/producers/python-bandit/main.go

```diff
@@ -87,6 +88,9 @@ type BanditResult struct {
 	MoreInfo        string   `json:"more_info"`
 	TestID          string   `json:"test_id"`
 	TestName        string   `json:"test_name"`
+	IssueCWE        struct {
+		ID int32 `json:"id"`
```

Comment:
why is this specifically an int32? the default int Golang type is already 32 bits long, using an int32 just makes everything more verbose when processing it since all the default Golang libraries return `int`

Reply:
The `Issue.Cwe` in `issue.pb.go` is `[]int32` - do u want me to change that?

Reply:
I didn't notice that we are using `int32`, I don't see a reason to honestly, but it's going to be a hassle to change the protos, so let's stick with it for the time being and we revisit it in the future.